### PR TITLE
Good practice in naming shared space 

### LINKED ISSUE #549: Shared group space - naming good practice
### What would you like to see added?

Give some guidelines on good names for group spaces

Qualities of a good name:

- Short
- Memorable
- Relevant
- Abstract

If there is any chance the shared space could serve more than one grant or research project, we recommend avoiding names related to a specific project. Instead, name it for something durable and robust, that won't easily change with time.

We recommend names like

- `my_name_lab`
- `blazerid_group`
- `core_facility_space`

### PR DESCRIPTION
**Pull request by Premas**
# Pull Request

## Overview
Added good practice to naming shared space.

## Proposed Changes
 Minor changes to naming shared space.

## Related Issues

Fixes #549

### CODE REVIEW

File: docs/data_management/storage.md [hunk 1]
```diff
@@ -59,10 +59,19 @@ Please provide the following information. Missing information can delay allocati
 - **Internal UAB Collaborator Organizations:** The name(s) of any other UAB organizations participating in the Shared Storage.
 - **External Collaborator Organizations:** The name(s) of any external organizations participating in the Shared Storage.
 - **Regulatory Requirements:** List any regulatory requirements or agencies affecting data to be stored in the space. Possibilities include, but are not limited to: IRB, EHR, HIPAA, PHI, FERPA.
-- **Name of Shared Storage:** Please give us a short, memorable name that is specific to your organization but general to your purpose.
+- **Name of Shared Storage:** Please give us a generic name but specific to your project/lab.
```
Comment (wwarriner) [inline docs/data_management/storage.md:62]: Remove "but", so it reads "generic name that is specific to your project/lab"

File: docs/data_management/storage.md [hunk 2]
```diff
@@ -59,10 +59,19 @@ Please provide the following information. Missing information can delay allocati
...
-    - For Cores we recommend a shortened version of the Core name.
+    - For Labs we recommend the format `<PI_blazerid>_lab`. Example: PI_blazerid_lab, PI_name_lab, PI_blazerid_group
+    - For Cores we recommend a shortened version of the Core name. Example: core_facility_space
+
+    <!-- markdownlint-disable MD046 -->
+    !!! Tip
+
+        - Keep the name short, memorable, and relevant.
+        - Use `underscores (_)` or `hyphen (-)` to separate words.
+        - To serve future projects, consider names that are reusable.
```
Comment (wwarriner) [inline docs/data_management/storage.md:73]: Let's change "reusable" to "generic"

**Review by wwarriner — CHANGES_REQUESTED** [review]

File: docs/data_management/storage.md (hunk 1 above)
Comment (Premas) [reply docs/data_management/storage.md]: Changes incorporated

File: docs/data_management/storage.md (hunk 2 above)
Comment (Premas) [reply docs/data_management/storage.md]: Changes incorporated.

**Review by wwarriner — APPROVED** [review]

**Pull request merged**